nav2_simple_commander/nav2_simple_commander/robot_navigator.py

```diff
@@ -231,6 +256,8 @@ def getPath(self, start, goal):
             return None
 
         return self.result_future.result().result.path
+    
```

Comment:
run `ament_flake8` locally and fix all the issues it brings up. CI was complaining about some styling inconsistencies

Reply:
Fixed it in commit b874c9a65d81679ccac08327565e355c8a0178b4. Now CI is failing for some other reason, please have a look at it.